src/components/cc-block/cc-block.js

```diff
@@ -160,7 +160,7 @@ export class CcBlock extends LitElement {
 
         ::slotted([slot='title']) {
           flex: 1 1 0;
-          color: var(--cc-color-text-strong);
+          color: var(--cc-color-text-primary-strongest);
```

Comment:
nitpick: Could we move this into another commit (along with modification on cc-button and cc-env-var-form and default-theme.css)